snmpwalk/manifest.json

```diff
@@ -1,18 +1,15 @@
 {
   "maintainer": "help@datadoghq.com",
-  "manifest_version": "0.1.0",
-  "max_agent_version": "6.0.0",
-  "min_agent_version": "5.6.3",
+  "manifest_version": "1.0.0",
   "name": "snmpwalk",
+  "display_name": "Snmpwalk",
```

Comment:
Is this the correct case?

Reply:
dunno, utility name is `snmpwalk` all lowercase. We capitalize the name elsewhere, see https://github.com/DataDog/integrations-core/blob/master/nfsstat/manifest.json#L6